### PR TITLE
Add workspace/executeCommand request

### PR DESCRIPTION
**Pull request by yaohaizh**
Signed-off-by: Yaohai Zheng <yaozheng@microsoft.com>

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -59,7 +59,12 @@ export namespace Commands {
      */
     export const APPLY_WORKSPACE_EDIT = 'java.apply.workspaceEdit';
 
-	/**
+    /**
+     * Execute Workspace Command
+     */
+    export const EXECUTE_WORKSPACE_COMMAND = 'java.execute.workspaceCommand';
```
Comment (testforstephen) [inline src/commands.ts]: please add command `java.execute.workspaceCommand` to the `activationEvents` list in package.json

Comment (akaroml) [reply src/commands.ts]: Echo here. This is to make sure the extension is properly activated before the command is triggered.

Comment (yaohaizh) [reply src/commands.ts]: Already fixed.

**Comment by fbricon** [comment]
@yaohaizh can you please solve the merge conflict so we can merge it?

**Comment by yaohaizh** [comment]
Resolved conflication

**Pull request merged**